.docker/docker-entrypoint.sh

```diff
@@ -17,7 +17,8 @@ if [ "$1" = 'apache2-foreground' ] || [ "$1" = 'bin/console' ] || [ "$1" = 'php'
     ## Check if variable DATABASE_HOST is set, if yes, we have database
     if [[ -v DATABASE_HOST ]]; then
         ## Wait until database connection is ready
-        until mysql -u $DATABASE_USER -h $DATABASE_HOST --password="$DATABASE_PASSWORD" -e "" ; do
+        # @todo needs update
+        until postgres -u $DATABASE_USER -h $DATABASE_HOST --password="$DATABASE_PASSWORD" -e "" ; do
```

Comment:
What whould be here? I haven't use postgres in CLI manner at all.

Reply:
me neither, i am really sceptic to this migration, to technology none of us both can maintain at good grade